config-model-api/src/main/java/com/yahoo/config/application/api/DeploymentInstanceSpec.java

```diff
@@ -251,6 +261,16 @@ public boolean deploysTo(Environment environment, RegionName region) {
         return zones().stream().anyMatch(zone -> zone.concerns(environment, Optional.of(region)));
     }
 
+    /** Returns the zone endpoint specified for the given region, or the default, or {@code null}. */
```

Comment:
Should be "or empty".

Reply:
Yes, thanks, forgot doc when I changed the method.